### PR TITLE
New file detailing how to add non-core container images

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

**Comment by JoanneOFlynn2018** [comment]
@leifmadsen I haven't added this to an assembly yet. Is it most suited to Completing the Service Telemetry Framework Configuration chapter? 

I will add a 13 wrapper to the include when I find the right spot.

File: doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="adding-container-images-to-the-undercloud_{context}"]
+= Adding {OpenStack} container images to the undercloud
+
+[role="_abstract"]
+If you synchronized container images to a local registry, you must create an environment file and include the paths to the container images.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc:27]:
```suggestion
If you synchronized container images to a local registry, you must create an environment file and include the paths to the container images.
```

Can you give any more context here?

File: doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+parameter_defaults:
+    DockerCollectdConfigImage: <image-registry-path>/rhosp{vernum}/openstack-collectd:latest
+  	DockerCollectdImage: <image-registry-path>/rhosp{vernum}/openstack-collectd:latest
+  	DockerMetricsQdrConfigImage: <image-registry-path>/rhosp{vernum}/openstack-qdrouterd:latest
+  	DockerMetricsQdrImage: <image-registry-path>/rhosp{vernum}/openstack-qdrouterd:latest
+----
++
+Replace `<image-registry-path>` with the path to the image registry.
+
+. To deploy collectd and QDR to your overcloud, include the ``/usr/share/local-container-images/container-images.yaml` environment file so {OpenStack} director can prepare the images. The following snippet is an example of how to include this environment file:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc:47]:
```suggestion
. To deploy collectd and QDR to your overcloud, include the `/usr/share/local-container-images/container-images.yaml` environment file so {OpenStack} director can prepare the images. The following snippet is an example of how to include this environment file:
```

File: doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="adding-container-images-to-the-undercloud_{context}"]
+= Adding {OpenStack} container images to the undercloud
+
+[role="_abstract"]
+If you synchronized container images to a local registry, you must create an environment file and include the paths to the container images.
+
+.Prerequisites
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc:29]:
```suggestion
.Prerequisites
```

Any prerequisites that the reader should perform first?

**Comment by leifmadsen** [comment]
> @leifmadsen I haven't added this to an assembly yet. Is it most suited to Completing the Service Telemetry Framework Configuration chapter?
> 
> I will add a 13 wrapper to the include when I find the right spot.

Yea I think that's as good as any place. This is an optional thing only if you're using a local image registry.

File: doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc (hunk 3 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc:29]: I don't think so. Only prerequisite I suppose would be to make sure you've synchronized the image to your local registry?

File: doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+parameter_defaults:
+    DockerCollectdConfigImage: <image-registry-path>/rhosp{vernum}/openstack-collectd:latest
+  	DockerCollectdImage: <image-registry-path>/rhosp{vernum}/openstack-collectd:latest
+  	DockerMetricsQdrConfigImage: <image-registry-path>/rhosp{vernum}/openstack-qdrouterd:latest
+  	DockerMetricsQdrImage: <image-registry-path>/rhosp{vernum}/openstack-qdrouterd:latest
+----
++
+Replace `<image-registry-path>` with the path to the image registry.
+
+. To deploy collectd and QDR to your overcloud, include the `/usr/share/local-container-images/container-images.yaml` environment file so {OpenStack} director can prepare the images. The following snippet is an example of how to include this environment file:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc:47]:
```suggestion
. To deploy collectd and {MessageBus} to your overcloud, include the `/usr/share/local-container-images/container-images.yaml` environment file so {OpenStack} {OpenStackInstaller} can prepare the images. The following snippet is an example of how to include this environment file:
```

**Review by leifmadsen — APPROVED** [review]

File: doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc (hunk 3 above)
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_adding-container-images-to-the-undercloud.adoc:29]:
```suggestion

```

**Review by leifmadsen — APPROVED** [review]

**Pull request merged**